mlshvch/hometask-3/rainfall.rb

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+EXIT_COMMAND = 'exit!'
+ERROR_MESSAGE = 'City name can not be blank!'
+
+def extract_data(strng)
```

Comment:
Does making a variable name one symbol shorter helps in any way? Let's use the names that are descriptive enough and can be easily readable. `strng` is neither of those. It does not speak about what kind of string it is or what information does it contain.